### PR TITLE
Add command name of the problematic command to exception messages in app commands

### PR DESCRIPTION
**Pull request by TheXorog**
Tested by causing the changed exceptions.

### CODE REVIEW

**Review by Lulalaby — APPROVED** [review]

File: DisCatSharp.ApplicationCommands/Workers/ApplicationCommandWorker.cs [hunk 1]
```diff
@@ -75,9 +75,9 @@ > ParseContextMenuCommands(Type type, IEnumerable<MethodInfo> methods, List<Comm
 
 			var parameters = contextMethod.GetParameters();
 			if (parameters.Length == 0 || parameters == null || !ReferenceEquals(parameters.FirstOrDefault()?.ParameterType, typeof(ContextMenuContext)))
-				throw new ArgumentException($"The first argument must be a ContextMenuContext!");
+				throw new ArgumentException($"The first argument of the command '{contextAttribute.Name}' has to be an ContextMenuContext!");
```
Comment (xMaxximum) [inline DisCatSharp.ApplicationCommands/Workers/ApplicationCommandWorker.cs:78]: shouldn't it be ".. a ContextMenuContext"?

Comment (Lulalaby) [reply DisCatSharp.ApplicationCommands/Workers/ApplicationCommandWorker.cs:78]: F

Comment (Lulalaby) [reply DisCatSharp.ApplicationCommands/Workers/ApplicationCommandWorker.cs:78]: well next time

Comment (TheXorog) [reply DisCatSharp.ApplicationCommands/Workers/ApplicationCommandWorker.cs:78]: Oopsies

**Pull request merged**